src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -336,65 +338,64 @@ export default {
 		}
 	},
 	apollo: {
-		preFetch(config, client, args) {
+		preFetch(_config, client, args) {
 			return client.query({
 				query: loanChannelPageQuery
-			}).then(({ data }) => {
-				// filter routes on route.param.category to get current path
+			}).then(async ({ data }) => {
+				// Filter routes on route.param.category to get current path
 				const targetedLoanChannelURL = _get(args, 'route.params.category');
-				// isolate targeted loan channel id
+
+				// Isolate targeted loan channel id
 				const targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, data);
-				// extract query
+
+				// Extract query
 				const pageQuery = _get(args, 'route.query');
 
-				return client.query({
-					query: loanChannelQuery,
-					variables: _merge(
-						{
-							ids: [targetedLoanChannelID],
-							limit: loansPerPage
-						},
-						fromUrlParams(pageQuery)
-					)
-				});
-			});
+				await preFetchChannel(
+					client,
+					// Access map directly since SSR doesn't have mixins available
+					loanChannelQueryMapMixin.data().loanChannelQueryMap,
+					targetedLoanChannelURL,
+					// Build loanQueryVars since SSR doesn't have same context
+					{ ids: [targetedLoanChannelID], limit: loansPerPage, offset: fromUrlParams(pageQuery).offset }
+				);
+			}).catch(e => logReadQueryError(e, 'LoanChannelCategoryControl preFetch loanChannelPageQuery'));
```

Comment:
I don't think we can use `logReadQueryError` in preFetch, since the error will just get swallowed up server side -- but I could be wrong

Reply:
The error gets logged to the server-side logs (verified locally). I can remove this if we don't actually access these logs. My assumption was that all of the logs were somewhere in a cloud logging instance.

Reply:
@eddieferrer Good to keep this log then or should I remove?

Reply:
Checked all usages of `preFetch`, and we generally don't log errors, so I removed the `catch`. While having a `catch` would be good defensive programming, there are many issues if the connections fail for the UI, so adding a `catch` here doesn't save the app from erroring out in that edge case.